### ISSUE TITLE
An (.)protolint.y(a)ml seems to be necessary?

### ISSUE THREAD

**Issue by Ilyes512** [plexsystems/vscode-protolint#24]
So protolint it self does not require a config file, but it seems this plugin does look for one by default. Even when you create one and its empty, it will complain about the config being empty...

No file results in:
```
protolint: not found config file by searching `<PATHS_IT_TRIES_TO_FIND_THE_CONFIG_IN>`
```

Empty file results in:
```
protolint: read <PATH>/.protolint.yml, but the content is empty
```

It won't complain if the file contains:

```yaml
lint:
```

**Comment by jpreese** [comment]
Does not having a configuration stop the extension from working at all? or is this just an annoying error message?

If it's the latter, it would probably make sense to just have the extension report a missing config as a debug/info so you know that it wasn't found and is using the defaults.

**Comment by leefernandes** [comment]
Is it possible to specify the path to `.protolint.yaml` with settings?

**Comment by jpreese** [comment]
At the moment, there isn't a setting to be able to set where the protolint config exists. It just uses the current workspace folder and traverses from there, trying to find a protolint config.

This issue is specific to being noisy when a config isn't found (but could be desirable). Are you needing to be able to specify a specific directory where the config is?

**Comment by jpreese** [comment]
@Ilyes512 this should be fixed in `0.7.0`
